logging/handlers.rst

```diff
@@ -0,0 +1,105 @@
+BufferHandler) in order to call Elasticsearch only once with a bulk push. For
+even better performance and fault tolerance, a proper ELK
+(https://www.elastic.co/what-is/elk-stack) stack is recommended.
+
+To use it, declare it as a service:
+
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/services.yaml
+        services:
+            Symfony\Bridge\Monolog\Handler\ElasticsearchLogstashHandler: ~
+
+    .. code-block:: xml
+
+        <!-- config/services.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:monolog="http://symfony.com/schema/dic/monolog"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/monolog
+                https://symfony.com/schema/dic/monolog/monolog-1.0.xsd">
+
+            <services>
+                <service id="Symfony\Bridge\Monolog\Handler\ElasticsearchLogstashHandler" />
+            </services>
+        </container>
+
+    .. code-block:: php
+
+        // config/services.php
+        use App\Logger\SessionRequestProcessor;
+        use Monolog\Formatter\LineFormatter;
+
+        $container->register('Symfony\\Bridge\\Monolog\\Handler\\ElasticsearchLogstashHandler');
+
+then reference it in monolog configuration:
+
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/prod/monolog.yaml
+        monolog:
+            handlers:
+                es:
+                    type: service
+                    id: Symfony\Bridge\Monolog\Handler\ElasticsearchLogstashHandler
+
+    .. code-block:: xml
+
+        <!-- config/packages/prod/monolog.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:monolog="http://symfony.com/schema/dic/monolog"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/monolog
+                https://symfony.com/schema/dic/monolog/monolog-1.0.xsd">
+
+            <monolog:config>
+                <monolog:handler
+                    name="es"
+                    type="service"
+                    id="Symfony\Bridge\Monolog\Handler\ElasticsearchLogstashHandler"
+                />
+            </monolog:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/prod/monolog.php
+        $container->loadFromExtension('monolog', [
+            'handlers' => [
+                'es' => [
+                    'type' => 'service',
+                    'id' => 'Symfony\\Bridge\\Monolog\\Handler\\ElasticsearchLogstashHandler',
```

Comment:
same question as above with the use statement.

Reply:
:+1: